troubleshooting/incrementally_configuring_images.md

```diff
@@ -0,0 +1,317 @@
+   ```
+
+   Note that this may have been logged in a different session within the IUF activity, so a
+   different IUF log directory may need to be examined.
+
+   The string starting with `argo/` is the key within the `config-data` bucket of the S3 object
+   which contains the log output from `sat bootprep`. Save this S3 key to an environment variable:
+
+   ```bash
+   LOGS_S3_KEY="argo/cos-products-20231031-gdm63-prepare-images-2rkds/cos-products-20231031-gdm63-prepare-images-2rkds-sat-wrapper-2664865581/main.log"
+   ```
+
+1. (`ncn-mw#`) Download the workflow SAT logs.
+
+   ```bash
+   cray artifacts get config-data "$LOGS_S3_KEY" main.log
+   ```
+
+1. (`ncn-mw#`) Set `CFS_CONFIGURATION_NAME`.
+
+   1. View the log and look at the end for the list of configurations that have been created.
+
+      ```json
+      {
+          "configurations": [
+              {
+                  "name": "ssi-compute-23.11.0-SSI-csm-1.5.0.beta37-10"
+              }
+          ]
+      }
+      ```
+
+   1. Set `CFS_CONFIGURATION_NAME` to the name of the [Configuration Framework Service (CFS)](../glossary.md#configuration-framework-service-cfs) configuration.
+
+      ```bash
+      CFS_CONFIGURATION_NAME="ssi-compute-23.11.0-SSI-csm-1.5.0.beta37-10"
+      ```
+
+   SAT may have generated multiple configurations, in which case it is up to the user to determine which is the desired configuration.
+   However if the default SAT `bootprep` file is being used, then the configuration name should include `compute`
+
+1. (`ncn-mw#`) Set `SESSION_NAME`.
+
+   This can be set to any value, but the following example steps assume this variable is set.
+
+   ```bash
+   SESSION_NAME="example-partial-image-configuration"
+   ```
+
+1. (Optional) Find the failed configuration layer.
+
+   If configuration of the image failed somewhere beyond the first configuration layer, and there is some confidence that the successful layers will continue to be successful,
+   then it is possible to save time by generating the first partial image with all of the successful layers.
+   Find the index of the of the failed configuration either by counting the number of successful configuration layers in the logs, or by comparing the name of the failed layer to the output of `cray cfs v3 configurations describe $CFS_CONFIGURATION_NAME`.
+   Indices for the configuration layers start at 0.
+
+## Generating a partially configured image
+
+1. (`ncn-mw#`) Set the configuration limit.
+
+   If this is the first CFS run of this procedure, this should be `0` to apply only the first layer.
+
+   ```bash
+   CONFIGURATION_LIMIT=0
+   ```
+
+   Optionally, this can instead be set to a comma-separated list of numbers starting at 0 when trying to apply all layers up to the failed layer. For example:
```

Comment:
```suggestion
   Optionally, this can instead be set to a comma-separated list of numbers starting at `0` when trying to apply all layers up to the failed layer. For example:
```